scaleway/resource_k8s_cluster_test.go

```diff
@@ -81,7 +81,22 @@ func testSweepK8SCluster(_ string) error {
 		}
 
 		for _, cluster := range listClusters.Clusters {
-			_, err := k8sAPI.DeleteCluster(&k8s.DeleteClusterRequest{
+			//remove pools
+			listPools, err := k8sAPI.ListPools(&k8s.ListPoolsRequest{Region: region, ClusterID: cluster.ID}, scw.WithAllPages())
```

Comment:
Put one argument per line to have something diff friendly

Reply:
ok arguments updated on last commit